frontend/js/i18n/de-lang.json

```
@@ -180,14 +180,14 @@
 	},
 	"settings": {
 		"default-site": "Standard-Seite",
-		"default-site-404": "Dead Seite",
+		"default-site-404": "Serverfehler: Host nicht gefunden",
```

```suggestion
		"default-site-404": "Hinweis auf nicht existente Seite",
```
maybe something like this?